pkg/TIPA.py

```
@@ -72,63 +72,61 @@ def run(self):
         for _ in range(self.num_workers):
             Worker(self.process_queue, self.lock).start()
 
+        self.capture_screenshots()
+
+    def capture_screenshots(self):
```

Ruff (0.8.0)

Missing return type annotation for public function `capture_screenshots`

Add return type annotation: `None`

(ANN201)